userguide/directoryservices.rst

```diff
@@ -264,18 +264,22 @@ Click the :guilabel:`REBUILD DIRECTORY SERVICE CACHE` button if a new
 Active Directory user needs immediate access to %brand%. This occurs
 automatically once a day as a cron job.
 
-.. note:: Active Directory places restrictions on which characters are
-   allowed in Domain and NetBIOS names, and limits the length of those
-   names to 15 characters. If there are problems connecting to the
-   realm,
-   `verify
-   <https://support.microsoft.com/en-us/help/909264/naming-conventions-in-active-directory-for-computers-domains-sites-and>`__
-   the settings do not include any disallowed characters. The
-   Administrator account password cannot contain the *$* character. If a
-   *$* exists in the domain administrator password,
-   :command:`kinit` will report a "Password Incorrect" error and
-   :command:`ldap_bind` will report an "Invalid credentials (49)"
-   error.
+Active Directory places restrictions on which characters are allowed in
+Domain and NetBIOS names, and limits the length of those names to 15
```

Comment:
```suggestion
 Domain and NetBIOS names, and limits the length of those names to fifteen
```
This isn't really an issue. Just a style preference. I've always used the words if the numbers are under a certain syllable count. The most important thing, however, is that it is consistent throughout the document. For example, either keep using digits for every number or use words for numbers that can be written in one word.

Reply:
I'll check again with Warren, but I think our current standard is to write numbers ten and less and use numerals for 11 and larger.

Reply:
Confirmed with Warren